src/libsync/account.cpp

```diff
@@ -174,6 +174,11 @@ QUrl Account::davUrl() const
     return Utility::concatUrlPath(url(), davPath());
 }
 
+QUrl Account::zsyncUrl() const
+{
+    return Utility::concatUrlPath(url(), QLatin1String("/index.php/apps/files/api/v1/zsync"));
```

Comment:
Wondering if this should be a separate app on the server instead.

Reply:
My personal view is no because the core changes aren't at all at the app level but in ChunkingUpload.php. The current request by @DeepDiver1975 and @PVince81 on servside is to replace this route with one part of dav, so this route will be something like: `remote.php/dav/files/$user/foo/bar.png?zsync`